package.json

```diff
@@ -422,6 +422,35 @@
           "inventory-storage.instances.item.put"
         ],
         "visible": true
+      },
+      {
+        "permissionName": "ui-inventory.item.create",
+        "displayName": "Inventory: Can view and create item",
+        "subPermissions": [
+          "ui-inventory.instance.edit",
+          "inventory.items.item.post"
+        ],
+        "visible": true
+      },
+      {
+        "permissionName": "ui-inventory.item.edit",
+        "displayName": "Inventory: Can view, create and edit item",
+        "subPermissions": [
+          "ui-inventory.item.create",
+          "inventory.items.item.put",
+          "inventory-storage.items.item.get",
+          "inventory-storage.items.collection.get"
```

Comment:
It seems like these two might belong under the above permission that includes with "Can _view_ ... item" Are you confident they belong here under "... and edit item" instead?

Reply:
Yes, I should replace it to view permission. I'll fix it. Thank you.